x-pack/plugins/triggers_actions_ui/public/application/lib/action_connector_api/action_connector_api.test.ts

```diff
@@ -5,23 +5,35 @@
  * 2.0.
```

Comment:
Since the `action_connector_api.ts ` file is removed, should we move these tests under the new files? (ex: `create.tests.ts`, `execute.test.ts`, etc)